core/src/test/java/org/testcontainers/junit/ParameterizedDockerfileContainerTest.java

```diff
@@ -26,39 +17,33 @@
 public class ParameterizedDockerfileContainerTest {
 
     @Parameterized.Parameters(name = "{0}")
-    public static Object[] data() {
-        return new Object[] { "alpine:3.2", "alpine:3.3" };
+    public static Object[][] data() {
+        return new Object[][] {
+                { "alpine:3.2", "3.2"},
+                { "alpine:3.3", "3.3"},
+                { "alpine:3.4", "3.4"}
+        };
     }
 
-    public ParameterizedDockerfileContainerTest(String baseImage) {
+    public ParameterizedDockerfileContainerTest(String baseImage, String expectedVersion) {
         container = new GenericContainer(new ImageFromDockerfile().withDockerfileFromBuilder(builder -> {
                 builder
                         .from(baseImage)
-                        .run("apk add --update nginx")
-                        .cmd("nginx", "-g", "daemon off;")
                         .build();
-            })).withExposedPorts(80);
+            })).withCommand("sleep", "30");
```

Comment:
how about `top` instead of sleep? Even with big delay number, sleep seems less reliable

Reply:
Good idea, will do.